// @unified-latex/unified-latex-lint/index.cjs
"use strict";
Object.defineProperty(exports, Symbol.toStringTag, { value: "Module" });
const index$8 = require("./rules/unified-latex-lint-argument-color-commands/index.cjs");
const index$7 = require("./rules/unified-latex-lint-argument-font-shaping-commands/index.cjs");
const index$6 = require("./rules/unified-latex-lint-consistent-inline-math/index.cjs");
const index$5 = require("./rules/unified-latex-lint-no-def/index.cjs");
const index$4 = require("./rules/unified-latex-lint-no-plaintext-operators/index.cjs");
const index$3 = require("./rules/unified-latex-lint-no-tex-display-math/index.cjs");
const index$2 = require("./rules/unified-latex-lint-no-tex-font-shaping-commands/index.cjs");
const index$1 = require("./rules/unified-latex-lint-obsolete-packages/index.cjs");
const index = require("./rules/unified-latex-lint-prefer-setlength/index.cjs");
const lints = {
  unifiedLatexLintArgumentColorCommands: index$8.unifiedLatexLintArgumentColorCommands,
  unifiedLatexLintArgumentFontShapingCommands: index$7.unifiedLatexLintArgumentFontShapingCommands,
  unifiedLatexLintConsistentInlineMath: index$6.unifiedLatexLintConsistentInlineMath,
  unifiedLatexLintNoDef: index$5.unifiedLatexLintNoDef,
  unifiedLatexLintNoPlaintextOperators: index$4.unifiedLatexLintNoPlaintextOperators,
  unifiedLatexLintNoTexDisplayMath: index$3.unifiedLatexLintNoTexDisplayMath,
  unifiedLatexLintNoTexFontShapingCommands: index$2.unifiedLatexLintNoTexFontShapingCommands,
  unifiedLatexLintObsoletePackages: index$1.unifiedLatexLintObsoletePackages,
  unifiedLatexLintPreferSetlength: index.unifiedLatexLintPreferSetlength
};
exports.lints = lints;
